### DISCUSSION TITLE
Activate window using F1 as shortcut (similar to iTerm)

### DISCUSSION THREAD

**Discussion by Warkanlock** [kovidgoyal/kitty#6253] [Q&A]
Hey everyone, I started to use kitty this weekend; an amazing project you have built here

Although one minor question regarding how to access the terminal most efficiently: is there a way to use a shortcut as in iTerm using a hotkey like F1?

I tried but failed miserably in macOS using skhd, shortcut extension, etc so I was just wondering if there's a way using a built-in feature for this

**Comment by kovidgoyal** [comment]
See https://github.com/kovidgoyal/kitty/issues/45
